rep-2004.rst

```diff
@@ -72,137 +78,123 @@ For example, it may be the case the tool which launches and verifies a ROS graph
 Package Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-*Note: bullets below that start with [ROS Core], will be the prescription for what we do in the core packages in order to meet the associated requirements*
-
 Requirements to be considered a 'Level 1' package:
 
-* Version Policy:
-
-  * Must have a version policy (e.g. ``semver``)
-  * Must be at a stable version (e.g. for ``semver`` that means have a version >= 1.0.0)
-  * Must have a strictly declared public API
-  * Must have a policy for API stability
-  * Must have a policy for ABI stability
-  * Must have a policy that keeps API and ABI stability within a released ROS Distribution
-  * [ROS Core] will use ``semver``, will maintain API and ABI stability according to ``semver`` and will be ABI (and therefore API) stable within a ROS distribution
-
-* Change Control Process:
-
-  * Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
-  * Must have peer review policy for all change requests (e.g. require one or more reviewer)
-  * Must have Continuous Integration (CI) policy for all change requests
-  * Must have documentation policy for all change requests
-  * [ROS Core]:
-
-    * All changes will go through a pull request
-    * All pull requests will require at least one reviewer who did not author the pr (package may choose to increase this number)
-    * All pull requests will be tested via CI, and on all tier 1 platforms (if applicable)
-    * Any required changes to documentation (API documentation, feature documentation, release notes, etc.) must be proposed before merging related changes
+#. **Version Policy:**
 
-* Documentation:
+   #. Must have a version policy (e.g. ``semver``)
+   #. Must be at a stable version (e.g. for ``semver`` that means have a version >= 1.0.0)
+   #. Must have a strictly declared public API
+   #. Must have a policy for API stability
+   #. Must have a policy for ABI stability
+   #. Must have a policy that keeps API and ABI stability within a released ROS Distribution
 
-  * Must have documentation for each "feature" (e.g. for ``rclcpp``: create a node, publish a message, spin, etc.)
-  * Must have documentation for each item in the public API (e.g. functions, classes, etc.)
-  * Must have a declared license or set of licenses
-  * Must have a copyright statement in each source file
-  * Must have a "quality declaration" document, which declares the quality level and justifies how the package meets each of the requirements
+#. **Change Control Process:**
 
-    * Must have a section in the repository's ``README`` which contains the "quality declaration" or links to it
-    * Must register with a centralized list of 'Level 1' packages, if one exists, to allow for peer review of the claim
+   #. Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
+   #. Must have `Developer Certificate of Origin (DCO) <https://developercertificate.org/>`_ on pull requests
```

Comment:
@wjwwood since this is new - is DCO only required in Level 1?

Reply:
I'm not sure. Any packages at lower quality levels could always implement DCO checking even if they're not aiming for level 1, but the question remaining is should be enforce it for lower packages.

I'll bring this up at the next ROS 2 meeting for discussion.